modules/monitoring-creating-alerting-rules-for-user-defined-projects.adoc

```diff
@@ -40,16 +40,18 @@ spec:
   - name: example
     rules:
     - alert: VersionAlert # <1>
-      expr: version{job="prometheus-example-app"} == 0 # <2>
+      for: 1m # <2>
+      expr: version{job="prometheus-example-app"} == 0 # <3>
       labels:
-        severity: warning # <3>
+        severity: warning # <4>
       annotations:
-        message: This is an example alert. # <4>
+        message: This is an example alert. # <5>
 ----
 <1> The name of the alerting rule you want to create.
-<2> The PromQL query expression that defines the new rule.
-<3> The severity assigned to the alert.
-<4> The message associated with the alert.
+<2> The duration for the alert to be active before firing it.
+<3> The PromQL query expression that defines the new rule.
+<4> The severity assigned to the alert.
+<5> The message associated with the alert.
```

Comment:
Could be more restrictive if it's useful. Ex: "The severity that [actor] assigns to the alert."